.run-tests-ci.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e # everything must pass
+
+envname="${1}"
+
+if [ -z "$envname" ]; then
+    echo "Must pass a label for test artifact file e.g. py27"
+    exit 1
+fi
+
+export PYTHONPATH="src"
+coverage run -m pytest \
+    -s \
+    --junitxml=build/pytest-$envname.xml \
+    src/tests src/integration_tests
```

Comment:
Can alternatively be defined in a `pytest.ini` file in the project root with:
```
[pytest]
testpaths = src/test src/integration_tests
```

Reply:
Ok, may move it there later, although when using it locally you may want to run just `src/tests` due to the heaviness of the `integration_tests`.